app/services/food_service.rb

```diff
@@ -0,0 +1,14 @@
+class FoodService
+
+  def self.search(query, page_size)
+    query.gsub!(' ', "%20")
```

Comment:
I commented this out and everything still worked (the URI generated was `/foods?utf8=✓&q=sweet+potato&commit=Search`). According to [W3C](https://www.w3.org/Addressing/URL/uri-spec.html), `+` and `%20` are both acceptable representation of blank space in query params.